torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -565,7 +568,11 @@ Node* insertCloneBeforeNode(
     newNode->addInput(orig_data);
     newNode->output()->setType(orig_data->type());
     newNode->insertBefore(node);
-  } else if (orig_data->type()->kind() == TypeKind::TensorType) {
+  } else if (
```

Comment:
This function seems to have a lot in common with `addDummyCloneToBlock` above, can we abstract and merge the common part?